### DISCUSSION TITLE
Get Signing Keys Via Graphql

### DISCUSSION THREAD

**Discussion by ParkerNC** [community/community#67134] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I am attempting to use the GitHub api to aggregate the signing keys for a large number of users. It would be much faster to use Graphql to perform this task. However, there does not appear to be an equivalent operation to the REST api users/ssh_signing_keys and users/gpg_keys in Graphql. Am I mistaken on this? If not, is there any intention to add these operations to Graphql?

**Comment by honestmk99** [comment]
Hello ParkerNC

As of my knowledge, the GitHub GraphQL API does not currently have equivalent operations to the REST API endpoints `users/ssh_signing_keys` and `users/gpg_keys`. These specific operations for retrieving SSH signing keys and GPG keys are not available in the GraphQL schema provided by GitHub.

However, GitHub continues to actively develop and expand its GraphQL API. While I don't have access to GitHub's future plans, it's possible that they may add support for these operations in the future. To stay updated on the latest developments, I recommend checking GitHub's official documentation and their GraphQL API changelog for any announcements or updates regarding these features.
In the meantime, if you require SSH signing keys and GPG keys for a large number of users, you may need to continue using the REST API endpoints `users/ssh_signing_keys` and `users/gpg_keys` to aggregate this information efficiently.

Good luck

Best

**Comment by 1atAlcone** [comment]
I'm trying evrything to make something new everyday than to be board watching  encoders copy captchas haha but I'm curious where do they create their code. I'm so serious. I'm new to Github am really interested to know what's inside of every technology geek. Hehe if you have something to suggest please do so. I will appreciate.
